src/test/java/myPackage/MyStepdefs.java

```diff
@@ -77,4 +77,52 @@ public void iCloseTheBrowser() {
         browser.close();
         browser.quit();
     }
+
+    @Given("^I am on the home page$")
+    public void iAmOnTheHomePage() {
+        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
+        browser = new ChromeDriver();
+        String test_url = "http://newtours.demoaut.com/";
```

Comment:
why not add the link to another file.. and put all links together so it is easier for future if we need to update links.

Reply:
makes sense. I will update my code.